ros/src/bag_parser/scripts/parse_bag_lei.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/env python
```

Comment:
why do we have two different `parse_bag` scripts?

Reply:
parse_bag.py is thomas's first data generation pipeline implementation
I cloned thomas's branch and improved his pipeline which is pars_bag_lei.py

Reply:
@liulei1984, do we have to commit both of these? seems like one is improved copy of the other.